OBJC_INTEROP.md

```diff
@@ -0,0 +1,124 @@
+# _Kotlin/Native_ interoperability with Swift/Objective-C
+
+This documents covers some details of Kotlin/Native interoperability with
+Swift/Objective-C.
+
+## Usage
+
+Kotlin/Native provides bidirectional interoperability with Objective-C.
+Objective-C frameworks and libraries can be used in Kotlin code if
+properly attached to the build (system frameworks are attached by default).
```

Comment:
"attached to" -> "integrated with", attachment assumes somewhat asymmetrical relation.

Reply:
But why this relation should be described as being symmetrical?
I mean, when importing a library to Kotlin code through `cinterop`, we don't import Kotlin code to the library.

We may use replace "attach" by "include" or "import".

Reply:
Imported or used is fine with me.